modules/effects/src/effects_feature_module.ts

```diff
@@ -1,20 +1,36 @@
-import { NgModule, Inject, Optional } from '@angular/core';
-import { StoreRootModule, StoreFeatureModule } from '@ngrx/store';
+import { NgModule, Inject } from '@angular/core';
+import { Store } from '@ngrx/store';
 import { EffectsRootModule } from './effects_root_module';
 import { FEATURE_EFFECTS } from './tokens';
+import { getSourceForInstance } from './effects_metadata';
+
+export const FEATURE_EFFECTS_INIT = '@ngrx/feature-effects/init';
+export type FeatureEffectsInit = {
```

Comment:
The reason why I created this is to be typesafe, see the effects test case for a "demo".